third_party/2.7/itsdangerous.pyi

```diff
@@ -0,0 +1,153 @@
+# Stubs for itsdangerous (Python 2.7)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from datetime import datetime
+from itertools import izip
+from typing import Any, Callable, IO, MutableMapping, Optional, Text, Tuple, Union
+
+PY2 = ...  # type: bool
+text_type = unicode
+int_to_byte = chr
+number_types = (int, long, float)
+
+bytes_like = Union[bytearray, str]
```

Comment:
Hmm, that's an interesting idea.  I forgot that `bytes` aliases to `str` in Python 2.  The main motivation for splitting things up is it actually gets quite complicated in Python 3 as to which methods can take strings and which ones have to take a `bytes_like` value.  I'll investigate unifying the stybs.